upgrade/1.0/Stage_4.md

```diff
@@ -0,0 +1,9 @@
+# Stage 4 - CSM Service Upgrades
+
+Run `csm-service-upgrade.sh` to deploy upgraded CSM applications and services:
+
+```bash
+ncn-m002# /usr/share/doc/csm/upgrade/1.0/scripts/upgrade/csm-service-upgrade.sh
+```
+
+Once the `Stage 4` service upgrade is complete then please proceed to [Stage 5](Stage_5.md)
```

Comment:
Change to:

Once the `Stage 4` service upgrade is complete, proceed to [Stage 5](Stage_5.md).